R/xgboost-development.R

```diff
@@ -1,17 +1,16 @@
 #' Compare predictive models, created on your data
 #'
-#' @description This step allows you to create an XGBoost model, based on
-#' your data.
+#' @description This step allows you to create an XGBoost classification model, based on
```

Comment:
Again, a few more words or another sentence would be awesome describing how/why multiclass should be used. Keeping in mind that since this only requires 2 (or more) classes, someone might use it to compete with an rf or lasso classification model.